testsuite/libffi.call/cls_dbls_struct.c

```diff
@@ -57,10 +57,10 @@ int main(int argc __UNUSED__, char** argv __UNUSED__)
 	CHECK(ffi_prep_closure_loc(pcl, &cif, closure_test_gn, NULL, code) == FFI_OK);
 
 	((void*(*)(Dbls))(code))(arg);
-	/* { dg-output "1.0 2.0\n" } */
+	/* { dg-output "1.0 2.0" } */
```

Comment:
Could you walk me through why this is needed?  Looking at the test, I don't see anything really wrong with your patch, but I also don't understand why it is needed.